web/server/vue-cli/src/views/Reports.vue

```diff
@@ -445,6 +463,14 @@ export default {
           this.reports = reports;
           this.loading = false;
           this.initalized = true;
+
+          reports.forEach(report => {
+            ccService.getSameReports(report.bugHash).then(sameReports => {
+              this.sameReports[report.bugHash] =
+                [ ...new Set(sameReports.map(r => r.reviewData.status)) ];
```

Comment:
```suggestion
              this.$set(
                this.sameReports, report.bugHash,
                [ ...new Set(sameReports.map(r => r.reviewData.status)) ]);
```

For more information see: https://v2.vuejs.org/v2/guide/reactivity.html#For-Objects